core/kernel/pm.c

```diff
@@ -8,6 +8,7 @@
 #include <kernel/pm.h>
 #include <mm/core_memprot.h>
 #include <string.h>
+#include <malloc.h>
```

Comment:
Unrelated change?

Reply:
Compiler complains about `realloc`

Reply:
Please put this in a separate commit in this PR. It may seem a bit picky but it improves the history.

Reply:
Isn't `realloc()` in `<stdlib.h>`?

Reply:
Agree since `<stdlib.h>` includes `<malloc.h>`